docker/Dockerfile

```diff
@@ -23,9 +23,16 @@ RUN echo "Building python environment: stage Jupyter" \
   && env-build-tool extend /conf/requirements-jupyter.txt ${py_env_path}
 
 COPY jupyter-extensions.txt /conf/
+# TAB_NAME -- name to appear in the browser tab
+ARG TAB_NAME="DEA Sandbox"
 RUN echo "Adding jupyter lab extensions" \
-  xargs -n1 jupyter labextension install --no-build < /conf/jupyter-extensions.txt \
-  && jupyter lab build \
+  && while read ext; do \
+      if [ -n "${ext}" ]; then \
+        echo "Installing ${ext}"; \
+        jupyter labextension install --no-build "${ext}"; \
+      fi; \
+     done < /conf/jupyter-extensions.txt \
+  && jupyter lab build --name="${TAB_NAME}" \
```

Comment:
I'm guessing this allows you to pass a list of extra labextensions to install somehow? can you please document how it works?

Reply:
List of extensions is now in `jupyter-extensions.txt` that was merged in previous PR, but there was a bug missin `&&` just before `xargs`, this replaces xargs with a while loop so that I can see "Installing <extension name>" in the build log.

Reply:
Added to readme

Reply:
I am customising the tab name with template override, its not needed here

Reply:
@pindge I assume it doesn't hurt either? It's useful to me to build local versions I can easily tell apart at a glance when testing locally.

Reply:
It doesn't affect the theme template, I am overriding the title field in the template. 
```

  <meta charset="utf-8">
  <title>{{ html_title }}</title>
  <meta name="viewport" content="width=device-width, initial-scale=1">
```